src/ott/solvers/linear/univariate.py

```diff
@@ -161,6 +163,7 @@ def __call__(
       self,
       prob: linear_problem.LinearProblem,
       return_transport: bool = True,
+      return_dual_vectors: bool = True,
```

Comment:
i would rather put this to `False` by default, I can only foresee few use cases where this is needed.